README.md

```diff
@@ -6,7 +6,7 @@
   <h1>SnappyMail</h1>
   <br>
 
-[![github-actions](https://github.com/the-djmaze/snappymail/workflows/docker/badge.svg)](https://github.com/the-djmaze/snappymail/actions/workflows/docker.yml)
+[![github-actions](https://github.com/the-djmaze/snappymail/actions/workflows/docker.yml/badge.svg)](https://github.com/the-djmaze/snappymail/actions/workflows/docker.yml)

```

Comment:
this updates to the new URL structure: https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge